docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -137,6 +137,26 @@ You can configure the `expose-externally` field with the following properties:
 ** `NodePortNodeName`: Hazelcast clients are inside the same network as the Kubernetes nodes, so clients can access a member by the hostname of its Kubernetes node.
 ** `LoadBalancer`: Hazelcast clients are outside the Kubernetes network and can access members via the the external IP assigned to the LoadBalancer service.
 
+WARNING: Your Kubernetes cluster should be able to assign public IPs to LoadBalancer services to connect to Hazelcast from outside Kubernetes.
+
+WARNING: If NodePorts are used (`discoveryServiceType: NodePort` or `memberAccess: NodePortExternalIP`), then the Kubernetes nodes should have access from outside Kubernetes. This includes both the node IPs and ports.
+
+=== Things to consider in Google Kubernetes Engine (GKE):
```

Comment:
```suggestion
=== Google Kubernetes Engine

If you are using Google Kubernetes Engine (GKE), consider the following to expose your Hazelcast cluster:
```

I think we should follow this structure for all the 'Things to consider' headings below.

Heading: Just the name of the environment
Leading sentence: Explain what the following points are for.

Reply:
Done 👍

Reply:
It looks like all the headings are still missing the leading sentence.

Also, we can remove the colons (`:`) from the headings.

Reply:
Sorry, I misinterpreted your comment. Updated now. The leading sentences are so similar, is it a problem?